action.yml

```diff
@@ -126,12 +129,19 @@ runs:
         fi
 
     - name: Configure nix access-tokens
-      if: inputs.disable-nix-access-token == 'false'
+      if: inputs.disable-nix-access-token == 'false' && github.server_url == 'https://github.com'
```

Comment:
This is more of a nice to have: If you are running this action on a GitHub Enterprise Server, configuring the `github.token` as nix access token to `github.com` makes no sense, so we can skip it, and at least make the unauthenticated access work.